components/director/Makefile

```diff
@@ -8,3 +8,15 @@ include $(SCRIPTS_DIR)/generic_make_go.mk
 VERIFY_IGNORE := /vendor\|/automock\|/testdata
 
 verify:: gqlgen check-gqlgen
+
+errcheck-local:
+	 GO111MODULE="on" errcheck -blank -asserts -ignoregenerated ./...
+
+test-local:
+	GO111MODULE=on go test ./...
+
+resolve-local: ;
```

Comment:
I think it's bad to have it empty, we can easily specify that resolving deps is `go mod download`. You can add a message that this project uses Go modules and this step is not necessary. Eventually the target will be removed, when we move all components to Go modules.

Reply:
Similarly with `ensure-local`